src/Converter/Standard.php

```diff
@@ -111,4 +124,15 @@ public function getPreviousConverter()
     {
         return $this->parentConverter;
     }
+
+    public function setLeftDelimiter($p_delimiter) {
```

Comment:
Can the delimiters change after the object is instantiated? It not, then I'd rather add them as (optional) constructor arguments.